cdap-cli/src/main/java/co/cask/cdap/cli/command/GetProgramRunsCommand.java

```diff
@@ -78,12 +78,13 @@ public void perform(Arguments arguments, PrintStream output) throws Exception {
     }
 
     Table table = Table.builder()
-      .setHeader("pid", "end status", "start", "stop")
+      .setHeader("pid", "end status", "start", "run", "stop")
```

Comment:
So, `starting time`, `start time`, `stop time`? That's what we use in the `RunRecord` class